docs/components/control/authenticate-with-callback.mdx

```diff
@@ -141,7 +141,7 @@ The `<AuthenticateWithRedirectCallback />` is used to complete a custom OAuth fl
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `afterSignInUrl` (deprecated) | `string` | Full URL or path to navigate to after successful sign in. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. **`signInFallbackRedirectUrl` has priority over the deprecated `afterSignInUrl`. Use `fallbackRedirectUrl` or `forceRedirectUrl` instead of  `afterSignInUrl`.** |
-| `afterSignUpUrl` | `string` | Full URL or path to navigate to after successful sign up. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. |
+| `afterSignInUrl` (deprecated) | `string` | Full URL or path to navigate to after successful sign in. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#deprecated) instead. **`signInFallbackRedirectUrl` has priority over the deprecated `afterSignInUrl`. Use `fallbackRedirectUrl` or `forceRedirectUrl` instead of  `afterSignInUrl`.** |
+| `afterSignUpUrl` (deprecated) | `string` | Full URL or path to navigate to after successful sign up. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#deprecated) instead. |
```

Comment:
I'm not sure these specific props are actually deprecated - @nikosdouvlis could you confirm?

Reply:
Nikos seems busy so I just removed these changes from the PR for now

Reply:
Hello everyone :) 
These are, indeed, deprecated. The `AuthenticateWithRedirectCallback` component accepts the same props as `Clerk.handleRedirectCallback` now!